src/clj/harja/kyselyt/paallystys.sql

```diff
@@ -13,13 +13,7 @@ SELECT
   kaasuindeksi,
   lahetetty,
   lahetys_onnistunut          AS "lahetys-onnistunut",
-  lahetysvirhe,
-  ypk.aikataulu_kohde_alku AS "kohde-alkupvm",
-  ypk.aikataulu_paallystys_alku AS "paallystys-alkupvm",
-  ypk.aikataulu_paallystys_loppu AS "paallystys-loppupvm",
-  ypk.aikataulu_tiemerkinta_alku AS "tiemerkinta-alkupvm",
-  ypk.aikataulu_tiemerkinta_loppu AS "tiemerkinta-loppupvm",
-  ypk.aikataulu_kohde_valmis AS "kohde-valmispvm"
```

Comment:
Nämä tarvittiin kartan infopaneeliin silloin kun ollaan päällystysilmoitukset-näkymässä. Nykyään näytetään kartalla tällöin samat kohdeosat kuin kohdeluettelossakin, joten ei tarvita näitä enää.